.travis.yml

```diff
@@ -68,3 +68,5 @@ script:
 after_success:
     - if [[ $COVERAGE = true ]]; then wget https://scrutinizer-ci.com/ocular.phar; fi
     - if [[ $COVERAGE = true ]]; then php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml; fi
+    - wget http://tnyholm.se/reporter.phar
+    - php reporter.phar build:upload
```

Comment:
Will the build still report an error if these steps fail?

Reply:
Yes. The build will get it status from the `script`. What we do afterwards is not important.